tests/unit/IonDecimalTest.js

```diff
@@ -53,6 +53,7 @@ define([
             '12300d-2'  : () => test('12300d-2', '12300', -2, 123, '123.00'),
             '123456.0'  : () => test('123456.0', '1234560', -1, 123456.0, '123456.0'),
             '1234560d-1': () => test('1234560d-1', '1234560', -1, 123456.0, '123456.0'),
+            '9007199254740991.000000': () => test('9007199254740991.000000', '9007199254740991000000', -6, 9007199254740991, '9007199254740991.000000'),
```

Comment:
Please add another test for an enormous negative number to also exercise the `if(cStr[0] === '-')` branch.

Reply:
Fasho im adding a few more edge case values as well.

Reply:
I believe this has been addressed.